py/kubeflow/testing/notebook_tests/job.yaml

```diff
@@ -0,0 +1,37 @@
+# A batch job to run a notebook using papermill.
+# The YAML is modified by nb_test_util.py to generate a Job specific
+# to a notebook.
+#
+# TODO(jlewi): We should switch to using Tekton
```

Comment:
As I understand this PR set is up to use Tekton so this should be fixed?

Reply:
Not quite. This is about how we run the notebook on the actual KF cluster. We are currently using a K8s job; we could potentially use a Tekton task if we install Tekton on KF clusters.

Reply:
I had misunderstood it, thanks for the clarification!